### ISSUE TITLE
`tuya-cli wizard` should output valid JSON for better usability and interoperability 

### ISSUE THREAD

**Issue by SidShetye** [TuyaAPI/cli#82]
At present `tuya-cli wizard` outputs an object like
```
[
  {
    name: 'Kitchen 1',
    id: '00000000000000000000',
    key: 'aaaaaaaaaaaaaaaa'
  },
  {
    name: 'Kitchen 2',
    id: '00000000000000000000',
    key: 'aaaaaaaaaaaaaaaa'
  }
]
```

but it would be far more useful to have it print out valid JSON instead (e.g. homebridge wants this JSON, tedious for >5 devices) like

```
[
  {
    "name": "Kitchen 1",
    "id": "00000000000000000000",
    "key": "aaaaaaaaaaaaaaaa"
  },
  {
    "name": "Kitchen 2",
    "id": "00000000000000000000",
    "key": "aaaaaaaaaaaaaaaa"
  }
]
```

The fix is simple, at wizard.js ~ line 93

from
```
console.log(devices.map(device => ({name: device.name, id: device.id, key: device.local_key})));
```
to 
```
let devs = devices.map(device => ({name: device.name, id: device.id, key: device.local_key}))
console.log(JSON.stringify(devs, null, 2));
```
or something similar

**Comment by codetheweb** [comment]
Great idea, happy to accept a PR for this. 👍

**Comment by milo526** [comment]
I believe this was already fixed in a recent commit right?

https://github.com/TuyaAPI/cli/commit/2bdd195a5479b581e0d2572a9da8febf8c7d5ff1

Specifically with the new `-s` option.

**Comment by codetheweb** [comment]
Yep, forgot to link this issue to https://github.com/codetheweb/tuyapi/issues/390.

Thanks for the reminder.

Included in v1.15.0 of @tuyapi/cli.
